### ISSUE TITLE
Documentation for gpt2-squad-qg-hl

### ISSUE THREAD

**Issue by rempel1234** [p208p2002/Transformer-QG-on-SQuAD#5]
Hi! How's it going? Is there any documentation on using this model? If not, could I write some, and request you merge it to this repo? Thanks!

**Comment by p208p2002** [comment]
@rempel1234 Yes, we surely lack some documents; it's very welcome to contribute 😊

**Comment by rempel1234** [comment]
[0001-add-documentation-for-GPT2.patch](https://github.com/p208p2002/Transformer-QG-on-SQuAD/files/15168388/0001-add-documentation-for-GPT2.patch)
Attached is some documentation for using the gpt2 model. Thanks!
